src/test/fuzz/mini_miner.cpp

```diff
@@ -118,9 +118,10 @@ FUZZ_TARGET_INIT(mini_miner_selection, initialize_miner)
     LIMITED_WHILE(fuzzed_data_provider.ConsumeBool(), 100)
     {
         CMutableTransaction mtx = CMutableTransaction();
-        const size_t num_inputs = 2;
-        const size_t num_outputs = fuzzed_data_provider.ConsumeIntegralInRange<size_t>(2, 5);
+        const size_t num_inputs = std::min(size_t{2}, available_coins.size());
```

Comment:
In bbb927739955b1588d94c5fd8ed73ddf5afde46c "fuzz: Fix mini_miner_selection running out of coin"

Is it possible that `available_coins` is empty and so `num_inputs = 0`? It may be useful to have an additional assert before the inputs are filled below.

Reply:
Wouldn't be an alternative to replace `LIMITED_WHILE(fuzzed_data_provider.ConsumeBool(), 100)` for `LIMITED_WHILE(!available_coins.empty(), 100)`?

Reply:
@brunoerg: I think that would work as well, although it would probably lead to smaller clusters in some of the fuzzing

Reply:
> Happy to add an assert, though

Maybe add this? `assert(!available_coins.empty());` above this line

Reply:
I added the assert suggested by @glozow